server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oms/enums/ProjectTypeEnum.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.onlineschemachange.oms.enums;
+
+/**
+ * @author yaobin
+ * @date 2023-12-27
+ * @since 4.2.4
+ */
+public enum ProjectTypeEnum {
```

Comment:
What is this "Project"? I mean it should be different from ODC's project, so better to add a prefix (Such as "Oms") to specify.

Reply:
Enum suffix is no need.

Reply:
okey, done, rename to OmsProjectType